Components/Slider/slider.css

```diff
@@ -0,0 +1,17 @@
+.slider-container {
+  gap: 10px;
+  align-items: center;
+}
+.slider-range {
+  appearance: none;
+  background-color: var(--silder-color);
```

Comment:
There is a typo in your variable naming. I think it should be `slider-color`

Reply:
Sure i will fix it